### PR TITLE
feat: social media sharing buttons

### PR DESCRIPTION
**Pull request by melniiv**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

File: hkm/templates/hkm/base.html [hunk 1]
```diff
@@ -42,6 +42,10 @@
     property="og:description"
     content="{% trans 'OGDescription' %}"
   />
+  <meta
+    property="pinterest:description"
+    content="{% trans 'OGDescription' %}"
```
[the remark this answers is not in the document]
Comment (nikomakela) [reply hkm/templates/hkm/base.html:47]: Should there be some actual content here instead?

**Review by nikomakela — COMMENTED** [review]
Please check the OGDescriptio.

**Review by nikomakela — APPROVED** [review]
The browser tests are failing too. I left some trivial suggestions, bt you can decide what you really want to do.

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

File: hkm/static/hkm/js/hkm-v2.js [hunk 2]
```diff
@@ -285,6 +285,103 @@ palikka
...
+      windowTitle = 'Facebook - ' + title;
+      sharerParams = '?u=' + href + '&p[images][0]=' + imageUrl;
+    } else if($(this).hasClass('share-tw')) {
+      sharerBaseUrl = 'https://twitter.com/share';
+      windowTitle = 'X - ' + title;
+      sharerParams = '?url=' + href;
+    } else if($(this).hasClass('share-pin')) {
+      sharerBaseUrl = 'http://pinterest.com/pin/create/button/';
+      windowTitle = 'Pinterest -';
+      sharerParams = '?url=' + href + '&media=' + imageUrl + '&description=' + title;
```
[the remark this answers is not in the document]
Comment (melniiv) [reply hkm/static/hkm/js/hkm-v2.js]: refactored

File: hkm/static/hkm/js/hkm-v2.js [hunk 3]
```diff
@@ -285,6 +285,103 @@ palikka
...
+    var windowTitle = '';
+    var sharerBaseUrl = '';
+    var sharerParams = '';
+
+    if($(this).hasClass('share-fb')) {
+      sharerBaseUrl = 'https://www.facebook.com/sharer/sharer.php';
+      windowTitle = 'Facebook - ' + title;
+      sharerParams = '?u=' + href + '&p[images][0]=' + imageUrl;
+    } else if($(this).hasClass('share-tw')) {
+      sharerBaseUrl = 'https://twitter.com/share';
```
[the remark this answers is not in the document]
Comment (melniiv) [reply hkm/static/hkm/js/hkm-v2.js]: fixed

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Kuvaselaamo branch is deployed to platta: https://kuvaselaamo-pr308.dev.hel.ninja** :rocket::rocket::rocket:

**Review by jannetalvio — APPROVED** [review]

**Pull request merged**